src/components/structures/LoggedInView.tsx

```diff
@@ -645,28 +636,13 @@ class LoggedInView extends React.Component<IProps, IState> {
                 />;
                 break;
 
-            case PageTypes.MyGroups:
-                pageElement = <MyGroups />;
-                break;
-
             case PageTypes.HomePage:
                 pageElement = <HomePage justRegistered={this.props.justRegistered} />;
                 break;
 
             case PageTypes.UserView:
                 pageElement = <UserView userId={this.props.currentUserId} resizeNotifier={this.props.resizeNotifier} />;
                 break;
-            case PageTypes.GroupView:
-                if (SpaceStore.spacesEnabled) {
-                    pageElement = <LegacyCommunityPreview groupId={this.props.currentGroupId} />;
```

Comment:
@nadonomy are we happy for groups links (be it from matrix.to or otherwise) to cease function entirely with no clear error/messaging?

Reply:
We can stub the page with a "communities are gone, long live spaces" thing if preferred, though I'd strongly recommend we steer away from anything which involves the API given Synapse is planning to remove it soonish.

Reply:
Handling this sounds prudent. Will carry discussion to here though: https://github.com/matrix-org/matrix-react-sdk/pull/8027#issuecomment-1067124119